src/compiler/scala/tools/nsc/settings/MutableSettings.scala

```diff
@@ -455,6 +455,7 @@ class MutableSettings(val errorFn: String => Unit)
     type T = Boolean
     protected var v: Boolean = false
     override def value: Boolean = v
+    def boolValue: Boolean = v
```

Comment:
This is a change from @retronym, with a comment "Avoid needless Boolean.unbox in hot code paths."

All of this PR is non functional changes, so the implicit function is not really a concern. I think this is a CPU vs memory trade off, an extra field on a handful of objects against N calls and M call sites

That said it is what I inherited and I have not looked into that the effect is in either the generated code or the runtime performance, but I am sure that @retronym can comment as to if this change is useful or not

What would be the preferred name if this access is kept?